docs/nx-cloud/reference/launch-templates.md

```diff
@@ -189,7 +189,10 @@ launch-templates:
         uses: 'nrwl/nx-cloud-workflows/v4/workflow-steps/cache/main.yaml'
         inputs:
           key: 'package-lock.json|yarn.lock|pnpm-lock.yaml'
-          paths: 'node_modules'
+          paths: |
+            ~/.npm
+            # or ~/.cache/yarn
+            # or .pnpm-store
```

Comment:
our `.pnpm-store` seems to be in `~/workspace/.pnpm-store` and not at the home folder level